bootstrap/16-files.sh

```diff
@@ -0,0 +1,9 @@
+if [ ! -z $CUSTOM_FILES ]; then
+    info "Copy ${CUSTOM_FILES} to ${R}"
+
+    if [ -d $CUSTOM_FILES ]; then
+        install_recur ${CUSTOM_FILES}
+    else
+        info "Directory does not exist. Skip step."
```

Comment:
The info is very confusing. Which directory? There is info above that `CUSTOM_FILES` is being copied to the chroot environment, but here you're saying that the step is skipping because some directory does not exist. I think the part should be totally reworked. For example,

```
if [ ! -z ${CUSTOM_FILES} ] && [ -d ${CUSTOM_FILES} ]; then
    info "copying ${CUSTOM_FILES} to ${R}"

    install_recur ${CUSTOM_FILES}
fi
```